src/conda_package_handling/_version.py

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Since this is an autogenerated file by versioneer I suggest we exclude it from black/isort